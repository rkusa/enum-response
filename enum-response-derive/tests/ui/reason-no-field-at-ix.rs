#[macro_use]
extern crate enum_response_derive;

#[derive(EnumResponse)]
enum Error {
    #[response(reason_field = 1)]
    Tuple(String)
}

