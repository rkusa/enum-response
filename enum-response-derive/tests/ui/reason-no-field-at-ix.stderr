error: proc-macro derive panicked
 --> $DIR/reason-no-field-at-ix.rs:4:10
  |
4 | #[derive(EnumResponse)]
  |          ^^^^^^^^^^^^
  |
  = help: message: [error(reason_field = 1)]: No tuple field at 1 found for Tuple

error: aborting due to previous error

